features/networking/pod.feature

```diff
@@ -190,3 +190,126 @@ Feature: Pod related networking scenarios
+    Given I use the first master host		
+    And I run commands on the host:
+      | ifconfig tun0 \| grep -w inet \| awk '{print $2}' |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_tun0_ip clipboard
+    
+    Given I select a random node's host
+    And I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And I have a pod-for-ping in the project
+    And the pod named "hello-pod" becomes ready
+    #Curl on Master's tun0 IP to make sure connections are blocked to MCS via tun0
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_tun0_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_tun0_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23892
+  @admin
+  Scenario: A pod cannot access the MCS via an egress router in http proxy mode
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    Given I have a project
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    #pod-for-ping will be a non-hotnetwork pod
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/egress-http-proxy/egress-http-proxy-pod.yaml |
+      | n | <%= project.name %>                                                                                   |
+    Then the pod named "egress-http-proxy" becomes ready
+    #Pod should not access MCS via an egress router pod
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23893
+  @admin
+  Scenario: A pod in a namespace with an egress IP cannot access the MCS
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    And evaluation of `node.name` is stored in the :egress_node clipboard
+    #add the egress ip to the hostsubnet
+    And the valid egress IP is added to the "<%= cb.egress_node %>" node
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project clipboard
```

Comment:
You use only one project ins scenario if I'm not missing something. No need to ceate a clipboard. Just use `project.name` instead of `cb.project` whenever you need that.

Reply:
Thanks, taken care of